index.html

```diff
@@ -1,107 +1,84 @@
-                If you like what you see and have a project you need coded,
-                don’t hesitate to contact me.
-              </p>
-            </div>
-            <div class="about-section-item links">
-              <a href="http://" class="lets-connect">LET’S CONNECT</a>
-              <ul class="social-media-icons">
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="twitter"
-                      src="./assets//twitter.png"
-                      alt="twitter"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#">
-                    <img
-                      class="link"
-                      src="./assets//Linkedin icon.png"
-                      alt="linkedin"
-                    />
-                  </a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="socialIcon"
-                      src="./assets/socialIcon.png"
-                      alt="social"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img class="git"   
-                          src="./assets/git.png" 
-                          alt="git"
-                  /></a>
-                </li>
-                <li class="individual-icons">
-                  <a href="#"
-                    ><img
-                      class="twohandicon"
-                      src="./assets/twohandicon.png"
-                      alt="my pub"
-                  /></a>
-                </li>
-              </ul>
-            </div>
-          </div>
-        </div>
-      </section>
-    </div>
-    <!-- desktop -->
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Porfolio Website</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="header-and-headline-section">
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo">Cobby Elson</a>
+        <button type="button" class="hamburger-btn">
+          <span class="toggler-icon top-bar"></span>
+          <span class="toggler-icon middle-bar"></span>
+          <span class="toggler-icon bottom-bar"></span>
+        </button>
+        <ul class="desktop-nav-items">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
```

Comment:
- [ ] These links have failed the accessiblitily test. That is because you have not labelled them as links. Please add the anchor tag to them. 👍🏾